internal/resource/statefulset.go

```diff
@@ -494,15 +494,30 @@ func (builder *StatefulSetBuilder) podTemplateSpec(annotations, labels map[strin
 				{
 					Name:  "copy-config",
 					Image: builder.Instance.Spec.Image,
+					SecurityContext: &corev1.SecurityContext{
+						RunAsUser: pointer.Int64Ptr(0),
+						Capabilities: &corev1.Capabilities{
+							Drop: []corev1.Capability{
+								// Remove default capabilities allowed by Docker except for CHOWN and FOWNER
+								"SETPCAP", "MKNOD", "AUDIT_WRITE", "NET_RAW", "DAC_OVERRIDE", "FSETID",
```

Comment:
Can this be inverted? I think Add takes precedence over Drop, i.e:
```go
Capabilities: &corev1.Capabilities{
  Drop: []corev1.Capability{
    "ALL",
  },
  Add: []corev1.Capability{
    "CHOWN",
    "FOWNER",
  },
}
```

Or are you keeping some other ones?

Reply:
Thank you @coro 🙂 Yes, that's better. I changed it.